qiskit/quantum_info/synthesis/two_qubit_decompose.py

```diff
@@ -544,6 +544,52 @@ def specialize(self):
         self.K2r = np.asarray(RYGate(k2rtheta)) @ np.asarray(RXGate(k2rlambda))
 
 
+class TwoQubitWeylEchoRZX(TwoQubitWeylDecomposition):
+    """Decompose two-qubit unitary in terms of echoed cross-resonance gates.
+    """
+
```

Comment:
```suggestion

    def __init__(self, unitary, backend):
        self._backend = backend
        super().__init__(unitary)

```
You will need to clean this suggestion up after committing.